test-apps/display-test-app/src/frontend/Title.ts

```diff
@@ -8,7 +8,7 @@ import { IModelConnection, InteractiveEditingSession } from "@bentley/imodeljs-f
 
 export function setTitle(imodel: IModelConnection) {
   let prefix = "";
-  if (OpenMode.ReadWrite === imodel.openMode)
+  if (OpenMode.ReadWrite === imodel.openMode && imodel.isBriefcaseConnection())

```

Comment:
I see a bunch of added checks in display-test-app on `isBriefcaseConnection` but AFAICT display-test-app still opens standalone connections, not briefcase. Do display-test-app's editing tools still work?

Reply:
StandaloneConnection is now a subclass of BriefcaseConnection

Reply:
`BriefcaseConnection` docs say it is "A connection to a [BriefcaseDb]($backend) on the backend." StandaloneConnection is-a BriefcaseConnection but it connects to a StandaloneDb which is-not-a BriefcaseDb, right?

What does `contextId` return for a StandaloneConnection? What do `pushChanges` and `pullAndMergeChanges` do?

Reply:
I just removed the internal class `StandaloneConnection` entirely, since I had to change the name of its only method anyway. I updated the documentation to be accurate, but since `StandaloneDb` is internal, its only mention is in the openStandalone method.

Reply:
I am still interested in what happens when I try to pull/push changes or ask for contextId.

Reply:
"Standalone" should not be a subclass of "Briefcase".

I bet GeoModeler is using StandaloneConnection.

Reply:
@scsewall I removed `StandaloneConnection` and instead made a new method `BriefcaseConnection.openStandalone` (GeoModeler will have to change their calls to that). 

This makes sense to me according to the existing documentation you wrote on `StandaloneDb`: 

"Standalone iModels are designed such that the API interaction between Standalone iModels and Briefcase iModels (those synchronized with iModelHub) are as similar and consistent as possible."

BTW, in this PR we now enforce that StandaloneDb will only open iModels with projectId=empty and briefcaseId=0.

Reply:
@kabentley My "consistent as possible" comment was really about being able to pass in an `IModelDb` to most functions and not caring whether it was actually a `BriefcaseDb` or a `StandaloneDb`. During the prior refactor, `StandaloneDb/Connection` was separated out from `BriefcaseDb/Connection` to make it clear that pushing, pulling and applying changes was not relevant in the standalone case. I don't want to undo that.

Another idea would be to have the notion of a `FileConnection` on the frontend with `BriefcaseFileConnection` and `StandaloneFileConnection` subclasses. A briefcase and a standalone are both files, but I have a hard time accepting that a standalone iModel is a briefcase.

Reply:
@pmconne see OpenStandalone.test.ts

Reply:
Calling it a briefcase when it has no connection to the hub and throwing exceptions if user tries to treat it as a briefcase seems wrong to me.

Reply:
the whole concept of "standalone" is marked @internal, and i'm not sure it will ever get used. 

But it *should* be thought of as "a briefcase that isn't **yet** connected to iModelHub". I don't want programmers have to know the difference between this less-than briefcase and a real briefcase - for all intents and purposes, they *are* briefcases. When someone tries to use the (few) operations that aren't supported, they should get an alert that says "upgrade to iModelHub." Complicating things for this special use case isn't worth it.